src/artifact.h

```diff
@@ -62,6 +62,17 @@ enum art_charge : int {
     NUM_ARTCS
 };
 
+enum art_charge_req : int {
+    ACR_NULL = 0, //No extra requirement
+    ACR_EQUIP,    //Must be worn/wielded as appropriate
+    ACR_SKIN,     //As ACR_EQUIP, plus must be only* thing on a bodypart (or no gloves if wielded)
+    ACR_SLEEP,    //Only while sleeping            ^*Anything with <50% coverage doesn't count
```

Comment:
This leaking comment might cause trouble for future.

Reply:
Fair enough, changed it.